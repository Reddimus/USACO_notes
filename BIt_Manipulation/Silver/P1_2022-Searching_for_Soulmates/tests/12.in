10
2609441040 3134524456
2422262343 1654484191
998488456 1748419857
2646104179 2034299854
3575678746 1443079204
4222491694 3513722252
1256283606 1742761110
2595977995 709945232
4013957133 1704796510
3980639134 2680688758
