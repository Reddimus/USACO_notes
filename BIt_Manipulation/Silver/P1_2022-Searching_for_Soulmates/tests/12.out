81
93
82
87
86
88
82
74
95
82
